public/_redirects

```diff
@@ -5,7 +5,7 @@ https://asyncapi.com/* https://www.asyncapi.com/:splat 301!
 # Docs
 /docs /docs/getting-started 301!
 /docs/tooling /docs/community/tooling 301!
-/docs/specifications/latest /docs/specifications/2.0.0 302!
+/docs/specifications/latest /docs/specifications/latest 302!
```

Comment:
Maybe add comment why you use this redirect, to this same page. I guess for cached redirect, yes?

Reply:
Also we use here `302` status, so you can remove it :)

- 301 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301
- 302 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302

Reply:
Yeah, I overlooked this part. Just removed it.